Spring-2019/stacks_queues/SOLUTIONS_2019Spring.md

```diff
@@ -36,21 +36,43 @@ If the problem is simple enough, remove this section. TODO :bug:
 
 TODO :bug:
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. PROBLEM 3 Validate Stack Sequences
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/validate-stack-sequences/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
```

Comment:
Suggested changes to below line, just so folks don't try to get clever with the interpretation of the question. LMK what you think.

Reply:
yeah that's probably a fair assumption to make, although I'm pretty sure our solution also accounts for recently popped items that are pushed again